src/Microsoft.Framework.Runtime/Internal/ProjectUtilities.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.Framework.Runtime.Internal
+{
+    public class ProjectUtilities
```

Comment:
Or we can make `Microsoft.Framework.Runtime.Internal` internally visible to `Microsoft.Framework.Runtime.Roslyn.Tests` and `Microsoft.Framework.PackageManager.Tests` additionally.